plugins/jobs/src/js/data/JobModel.ts

```diff
@@ -176,21 +176,15 @@ export const resolvers = ({
       );
       const count$ = jobsInNamespace$.map(jobs => jobs.length);
 
-      return (
-        jobsInNamespace$
-          .map(jobs => filterJobsById(jobs, filter))
-          //TODO We need to remove the N + 1 query
-          // https://jira.mesosphere.com/browse/DCOS-38201
-          .map(jobs => jobs.map(job => fetchJobDetail(job.id)))
-          .switchMap(observable$ => Observable.combineLatest(observable$))
-          .map(requests => requests.map(JobTypeResolver))
-          .map(jobs => sortJobs(jobs, sortBy, sortDirection))
-          .combineLatest(count$, (jobs, totalCount) => ({
-            filteredCount: jobs.length,
-            totalCount,
-            nodes: jobs
-          }))
-      );
+      return jobsInNamespace$
```

Comment:
since there is no n+1 madness anymore, we should be able to simply do 
```
responses$
  .map(jobs => JobConnectionTypeResolver(jobs))
  .map(jobConnection => sortJobs(jobConnection, sortBy, sortDirection);
```
which then contains all the needed information, so no mapping over all jobs (here) and `combineLatest` necessary anymore.